tests/actions.go

```diff
@@ -195,8 +195,8 @@ type OperatorActions interface {
 	GetNodeMap(info *TidbClusterConfig, component string) (map[string][]string, error)
 	TruncateSSTFileThenCheckFailover(info *TidbClusterConfig, tikvFailoverPeriod time.Duration) error
 	TruncateSSTFileThenCheckFailoverOrDie(info *TidbClusterConfig, tikvFailoverPeriod time.Duration)
-	DeletePDDataThenCheckFailover(info *TidbClusterConfig, tikvFailoverPeriod time.Duration) error
-	DeletePDDataThenCheckFailoverOrDie(info *TidbClusterConfig, tikvFailoverPeriod time.Duration)
+	ReplacePDThenCheckFailover(info *TidbClusterConfig, tikvFailoverPeriod time.Duration) error
+	ReplacePDThenCheckFailoverOrDie(info *TidbClusterConfig, tikvFailoverPeriod time.Duration)
```

Comment:
I use changing PD image to simulate the failover, deleting PD data seems not stable here.

Reply:
Delete PD data is the case that matters, it is not same with changing PD image.

That may be the problem that we should resolve if it is not stable.

Reply:
got it, I would use Delete PD data back here soon.